src/components/layout/footer.module.css

```diff
@@ -0,0 +1,46 @@
+.nav {
+  display: flex;
+  flex-direction: row;
+  background-color: #122a50;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0% 2.5%;
+  box-shadow: 10px 21px 25px -7px rgba(0, 0, 0, 0.5);
+  height: 20em;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.logo {
+  width: 15em;
+  height: 15em;
+}
+
+.logo:hover {
+  cursor: pointer;
+}
+
+.list {
+  list-style-type: none;
+  border-right: 0.2em solid #fff;
+  text-align: right;
+  padding-right: 0.5em;
+}
+
+.listItem {
+  color: #fff;
+  padding-right: 0.5em;
+  padding: 1em 0.5em 1em 0;
+}
+
+a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 1.2em;
+}
+
+a:hover {
+  cursor: pointer;
+  color: #f35c3c;
+}
```

Comment:
Just a suggestion: instead of using class names like .list, .a, .nav, use more descriptive names like .footerList, .footerButton, .footerNav.

Reply:
Will remember for next time.